features/correction/service.py

```
@@ -0,0 +1,215 @@
+                correction["job_title"],
+                correction["job_description"],
+                correction.get("company_insight") or "",
+                portfolio_output,
+                correction.get("emphasis_points") or "",
+            )
+            if hasattr(result, "model_dump"):
+                result_dict = result.model_dump()
+            elif isinstance(result, dict):
+                result_dict = result
+            else:
+                raise ValueError("첨삭 결과 형식이 올바르지 않습니다.")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)